src/common/serverinfo.ts

```diff
@@ -61,18 +70,15 @@ function getServerInfo(connection: Connection): Promise<GetServerInfoResponse> {
   })
 }
 
-function computeFeeFromServerInfo(cushion: number,
-  serverInfo: GetServerInfoResponse
-): string {
-  return (new BigNumber(serverInfo.validatedLedger.baseFeeXRP)).
-    times(serverInfo.loadFactor).
-    times(cushion).toString()
-}
-
-function getFee(connection: Connection, cushion: number): Promise<string> {
-  return getServerInfo(connection).then(serverInfo => {
-    return computeFeeFromServerInfo(cushion, serverInfo)
-  })
+async function getFee(this: RippleAPI, cushion?: number): Promise<string> {
+  if (cushion === undefined) {
+    cushion = this._feeCushion
+  }
+  if (cushion === undefined) {
```

Comment:
Any reason to do it this way rather than the previous code (`cushion = this._feeCushion || 1.2`)?

Reply:
yea, if `this._feeCushion` is set to `0`, it would get overwritten to `1.2`.

I'm unfamiliar with the concept though, so maybe that's not something anyone would do :) but it's good to be safe

Reply:
Transactions probably won't succeed with a feeCushion < 1.0 since the result may be less than the base fee. But it's still a good idea to allow developers do what they want - it's possible they have a reason for it.